presto-main/src/test/java/io/prestosql/sql/planner/TestPredicatePushdown.java

```diff
@@ -57,6 +57,17 @@
         super(ImmutableMap.of(ENABLE_DYNAMIC_FILTERING, "true"));
     }
 
+    @Test
```

Comment:
@martint 
i think the test should be in the planner, 
is it a good enough place?